lib/dom.js

```diff
@@ -130,6 +130,70 @@ var INVALID_MODIFICATION_ERR = (ExceptionCode.INVALID_MODIFICATION_ERR = ((Excep
 var NAMESPACE_ERR = (ExceptionCode.NAMESPACE_ERR = ((ExceptionMessage[14] = 'Invalid namespace'), 14));
 var INVALID_ACCESS_ERR = (ExceptionCode.INVALID_ACCESS_ERR = ((ExceptionMessage[15] = 'Invalid access'), 15));
 
+// compareDocumentPosition bitmask results
+var DOCUMENT_POSITION_DISCONNECTED = 1;
+var DOCUMENT_POSITION_PRECEDING = 2;
+var DOCUMENT_POSITION_FOLLOWING = 4;
+var DOCUMENT_POSITION_CONTAINS = 8;
+var DOCUMENT_POSITION_CONTAINED_BY = 16;
+var DOCUMENT_POSITION_IMPLEMENTATION_SPECIFIC = 32;
```

Comment:
It looks like they should be available on the Node interface, just like the values for `nodeType`:
https://dom.spec.whatwg.org/#ref-for-dom-node-document_position_disconnected
Which should be helpful when adding tests.